test/e2e/sshsecrets_test.go

```diff
@@ -89,7 +89,9 @@ func assertSecretValueUpdated(ctx context.Context, t *testing.T, cfg *envconf.Co
 
 	paas := getPaas(ctx, "sshpaas", t, cfg)
 	paas.Spec.SshSecrets = map[string]string{"ssh://git@scm/some-repo.git": encrypted}
-	paas.Spec.Capabilities.SSO.SshSecrets = map[string]string{"ssh://git@scm/some-other-repo.git": encrypted}
+	SSO := paas.Spec.Capabilities["sso"]
+	SSO.SshSecrets = map[string]string{"ssh://git@scm/some-other-repo.git": encrypted}
```

Comment:
This way you're not updating the value of the struct appearently.

Reply:
Ha, thanks. I will uncomment line 94 and check again...